ros/src/twist_controller/dbw_helper.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
+#-------------------------------------------------------------------------------
+# Author: Kostas Oreopoulos <kostas.oreopoulos@gmail.com>
+# Date:   28.08.17
+#-------------------------------------------------------------------------------
+"""
+A collection of helper functions for dbw node
+"""
+# pylint: disable=import-error
+from math import sqrt, cos, sin
+import numpy as np
+import matplotlib.pyplot as plt
+import tf
```

Comment:
name 'tf' may be misleading when we start using tensorflow

Reply:
its the name of the package. its not mine :)

Reply:
besides you will not be using it in that same file

Reply:
ok, lets leave it :)

Reply:
Well, that's how they named the package. Not much can be done about it. Also, I believe they were first ;)